string class_name
sensor_msgs/Image mask
